apps/server/src/modules/engine/registry.ts

```diff
@@ -86,6 +92,9 @@ async function createEngine(engine, defaultFlogoDescriptorPath, skipBundleInstal
     await installResourceTypes(engine, defaultResourceTypes);
     await engine.installContribBundle(contribBundlePath);
     await tempInstallSimulatorDeps(engine.getProjectDetails());
+    if (bundleEngineConfig) {
```

Comment:
@fcastill I just realized, this should be moved to the top immediately after creating the engine. As bundling the config file and installing the contribution from bundle file are not related to each other. So setting the `skipBundleInstall` to false should **not** stop bundling the engine configuration. What do you think?

Reply:
true, I didn't see that, same for install resource types, I guess

Reply:
I mean something like:
```
if (!skipBundleInstall) {
  // await install bundle
}
await installResourceTypes();
if (bundleEngineConfig) {
  // await update engine config
}
```

which makes me wish the options were `skipContribBundle`/`skipEngineConfig` or `useContribBundle`/`useEngineConfig`

Reply:
`useContribBundle` `useEngineConfig` sounds good !! 

I think skipping `installResourceType` is apt there as while generating a binary from UI we do not want to install all resource types. I think we can club both resource type installation and contribution bundle installation with `useContribBundle` flag.

I will implement following for now: 
1. change the names of the flags
2. instead of returning out of the function I will do:
```
if (useContribBundle) {
  // await install bundle
await installResourceTypes();
}
if (useEngineConfig) {
  // await update engine config
}
```